src/controllers/queryRunner.ts

```diff
@@ -364,6 +369,22 @@ export default class QueryRunner {
         return outputString;
     }
 
+    private sendBatchTimeMessage(batchId: number, executionTime: string): void {
+        // get config copyRemoveNewLine option from vscode config
+        let config = this._vscodeWrapper.getConfiguration(Constants.extensionConfigSectionName);
+        let showBatchTime: boolean = config[Constants.configShowBatchTime];
+        if (!!showBatchTime) {
```

Comment:
Probably not. We used it elsewhere in these cases but given the config should give back a sensible value for the default I can skip it.